lib/diplomat/kv.rb

```diff
@@ -112,6 +114,7 @@ def put(key, value, options = nil)
         url += check_acl_token
         url += use_cas(@options)
         url += dc(@options)
+        url += acquire(@options)
```

Comment:
Moreover, this functionality is already added via `Diplomat::Lock.acquire` (just in case you were unaware); however, I don't see the harm in adding it here as well.